### ISSUE TITLE
Update multimedia section of accessibility guidance (partial)

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual#928]
## What

After a query via the service manual Slack channel, Rich Kelly has identified that our guidance on video audio, captions, transcripts needs updating. 

## Tasks
- [x] Sara to review Rich's draft guidance 
- [x] Rich to take the updated draft to Accessibility Working Group - with the other note below.
- [x] AWG to approve/comment
- [x] Sara and Rich to iterate
- [ ] Sara to publish updated guidance

**Comment by sarawilcox** [comment]
Note: put this to AWG too: <h2 id="pdfs">Use HTML rather than PDFs</h2>
<p class="nhsuk-body-s">For: <a href="/accessibility/content">Content</a>, <a href="/accessibility/testing">Testing</a></p>
<p>Avoid using  as they are not accessible. If you must use a PDF, make sure the content is also available in HTML form.</p>
<p>Avoid using PDFs as they are not accessible. If you must use a PDF, make sure the content is also available in HTML form.</p>
<p>Read more about <a href="/content/pdfs-and-other-non-html-documents">PDFs and other non-HTML documents in the content style guide</a>.</p>

**Comment by sarawilcox** [comment]
Latest draft, sent back to Rich. 
![Screenshot 2021-02-24 at 11 23 30](https://user-images.githubusercontent.com/45792959/108995438-07eb5500-7695-11eb-904a-a3ada5271372.png)

**Comment by sarawilcox** [comment]
Amelia will take the bit about 

> Note: put this to AWG too:
> 
> ## Use HTML rather than PDFs or other non-HTML documents
> For: [Content](/accessibility/content), [Testing](/accessibility/testing)
> 
> Avoid using PDFs or other non-HTML documents as they are not accessible. If you must use them, make sure the content is also available in HTML form.
> 
> Read more about [PDFs and other non-HTML documents in the content style guide](/content/pdfs-and-other-non-html-documents).

Amelia will take this bit to the next AWG meeting for approval before we amend the accessibility guidance partial.

**Comment by sarawilcox** [comment]
Put in review column pending AWG approval.

**Comment by cjforms** [comment]
A small cautionary note: the 'don't use .PDF' advice seems to have been misinterpreted by some folks as 'it's better to use open document format than PDF for forms'.  

Might be good to include a line about forms being a special case - still better as HTML than as PDF, but PDF is likely to be better than NOT having the form available.

**Comment by sarawilcox** [comment]
Hi Caroline, the latest guidance will say this: 

> If you're designing content that people will read on a screen, create it as structured web pages in HTML.
> If you want people to edit the content in a document, publish it in an open document format.

But for forms (i.e. anything used for "business purposes"), it will say the following: 

> ### Before 23 September 2018
> It's OK to keep non-HTML documents, including PDFs, created before 23 September 2018 as long as you're no longer using them for business processes, such as forms.
> 
> ### After 23 September 2018
> If you created or still use any non-HTML documents for business processes after 23 September 2018, you should do 1 of the following:
> 
> - keep the content up to date and replace them with an HTML page or give users an HTML alternative
> - delete them

**Comment by cjforms** [comment]
No, unfortunately I disagree with this. 

For example, last week I got a paper letter from my GP enclosing a poor photocopy of a paper form. IF the form had been available as an accessible .pdf on a website or as an email attachment then it would be much more convenient and accessible.

A Defra colleague has been doing investigations of using open document format forms, and they have severe limitations to the point of being unusable for anyone who tries to open them in an ordinary word processor such as Word. 

I believe it is NOT realistic to tell people that they must publish in HTML for a vast quantity of low-use forms, forms generated by people without access to a digital team, or teams who are making a determined effort to deal with higher-priority processes. 

I think we need to accept that accessible PDF is a useful interim format that is more accessible than paper, which is what will be used instead.

**Comment by cjforms** [comment]
If you have research on how to create good forms in open document format that can be used by people who only have standard word processing software such as Microsoft Word or Google Docs, then please share it. I'm very happy to have my mind changed about this but so far have been unable to find any user research other than the work done by my Defra colleague.

**Comment by sarawilcox** [comment]
Thanks @cjforms. We have now published our updated guidance: https://service-manual.nhs.uk/content/pdfs-and-other-non-html-documents/

Can I check? It sounds like your concern is that people will use ODFs rather than PDFs or HTML? 

We're saying that, for forms, we expect people to use HTML. We don't recommend ODFs as an alternative to forms. But perhaps the page isn't clear enough? When we mention ODFs at the top of the page, it's about editing documents and I understood 'documents' to mean collaborative documents, rather than forms: https://www.gov.uk/government/publications/open-standards-for-government/sharing-or-collaborating-with-government-documents. Maybe we need to make that clearer.

I'm seeing a fair amount of GOV.UK chat about problems with ODFs at the moment. I haven't seen people using ODFs in the NHS though and we don't have any research on this.

We've been taking advice from Alistair Duggin (accessibility expert). If people create PDFs correctly and save them in PDF/A (archiving) format, they *can* meet WCAG 2.1 AA. But he says there is no guarantee that doing this will make them meet the accessibility needs of our users in the way that HTML can. That's why we're steering people away from PDFs and other non-HTML docs in general. You say that it's not realistic to expect everyone to replace their old PDFs but that is the expectation now and we know that teams use this page to persuade stakeholders to go down the HTML route. 
 
We do still allow for PDFs in some rare cares - there are some examples on the page - but that's as well as an HTML page.

**Comment by cjforms** [comment]
Exactly. I only recently learned about the problems of using ODFs rather than PDFs or HTML for forms.

Of course I'm in favour of using HTML when that's feasible, but I'm definitely afraid that colleagues will interpret this guidance as 'you can't publish a .PDF form' and therefore decide not to publish, or to restrict their choice to paper (which is highly inaccessible for some people, although I still defend paper as an acceptable choice for others). 

I agree with Alistair. In fact, I think we all agree. Only I don't think the advice makes it clear. 

Here's what we agree on:
1. If your form is currently only on paper, then you must make an accessible version as well.
2. If you can, use an HTML form because this is the easiest to make accessible.
3. If you can't yet make an HTML form, then an accessible PDF is an acceptable option in the short term.

Can we make the guidance have a specific section on forms that aligns with that?

**Comment by cjforms** [comment]
By the way, I don't know of any research anywhere on ODT forms apart from the investigations done by my colleague at Defra. I'm hunting.

**Comment by sarawilcox** [comment]
Thanks @cjforms. I'm going to copy your above message into the PDF issue. I won't be able to take it forward immediately, but will create a ticket in our internal backlog to review over coming weeks.

**Comment by cjforms** [comment]
Thanks, that's great. I only found out about this problem a couple of weeks ago from my colleague Martin Glancy at Defra. I think we'll be working on it for a while - he may comment on this thread, too, and I'll alert him to the other one as well.

**Comment by sarawilcox** [comment]
Do you know if Martin is in touch with the GOV.UK content community chats about this on Basecamp? Or do you have a content designer with access to Basecamp chat?

**Comment by sarawilcox** [comment]
Prototype updated for video content. Rich Kelly will now present it to the Accessibility Working Group for approval. 

![Screenshot 2021-03-05 at 12 05 29](https://user-images.githubusercontent.com/45792959/110449007-a6c27900-80b9-11eb-9c26-e5a44e223399.png)
![Screenshot 2021-03-05 at 12 05 38](https://user-images.githubusercontent.com/45792959/110449031-ade98700-80b9-11eb-8ff2-5e85dae44f9a.png)

**Comment by sarawilcox** [comment]
Linked to accessibility guidance epic: https://github.com/nhsuk/nhsuk-service-manual/issues/669

**Comment by sarawilcox** [comment]
Copy iterated with Rich Kelly, taking on board comments from members of the Accessibility Working Group and Alistair Duggin. Rich will run latest version by them again.

**Comment by sarawilcox** [comment]
Iterated content with Rich. Ellen to run by deaf awareness group to check no issues with content. If OK, it's ready to publish.

We'll double check it with AWG at next meeting.

**Comment by sarawilcox** [comment]
Comment from Alistair:
Add a note to audio description to say:
If the video says the same thing as the page content, you do not need an audio description, but you must make it clear that the video is an alternative to the text content.

**Comment by sarawilcox** [comment]
PR ready for review. Rich Kelly and a content designer to proof the content in staging environment.

**Comment by sarawilcox** [comment]
All proofed. Ready to release.
